lib_nbgl/src/nbgl_use_case.c

```diff
@@ -1528,7 +1539,8 @@ static uint8_t nbgl_useCaseGetNbPagesForContent(const nbgl_content_t *content,
 
 static uint8_t nbgl_useCaseGetNbPagesForGenericContents(
```

Comment:
I thought static function should not start by `nbgl_` :thinking:

Reply:
I've not changed it, it was already like this...

Reply:
I know :stuck_out_tongue_winking_eye: